contracts/cm-v1.yaml

```diff
@@ -1654,4 +1712,37 @@ components:
                   enum: [HIU, HIP]
                 id:
                   type: string
-                  example: 100005
\ No newline at end of file
+                  example: 100005
+    HipInitiatedLinkRequest:
+      type: object
+      properties:
+        requestId:
+          type: string
+          format: uuid
+        timestamp:
+          $ref: '#/components/schemas/TimestampSchema'
+        cmId:
+          type: string
+        patient:
+          type: object
+          properties:
+            id:
```

Comment:
rename id to "referenceNumber". also need a "display" string. this is same as self initiated linking.

Reply:
done

Reply:
done